Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:22-alpine
```

Comment:
Can you update to use `FROM: node:23-alpine` ?

Reply:
`node:alpine` works :+1: Can it be merged soon?